INSTALL.md

```diff
@@ -22,6 +22,84 @@ Agda see:
 
 https://agda.readthedocs.io/
 
+cabal v2-install instructions
+=============================
+
+cabal `v2-build` is a new operating mode, which makes projects
+not interfere with each other. To download and compile the development
+version Agda with a `v2-build`, you need to download recent
+version of [`cabal-install`](https://www.haskell.org/cabal/download.html).
```

Comment:
How recent does this have to be?

Reply:
cabal-install-2.4

Reply:
Can you please add this so one knows whether an update is necessary or not

Reply:
Added